sphinx/source/docs/user_guide/compat.rst

```diff
@@ -9,32 +9,28 @@ you to use the most appropriate tool for each task.
 BokehJS
 -------
 
-Bokeh generates JavaScript, and so Bokeh output can be combined with a
-wide variety of JavaScript libraries, such as `React`_. Listing
-such libraries is beyond the scope of this document; it's best just to
+Bokeh generates JavaScript. This makes its output fully compatible with
+a wide variety of JavaScript libraries, such as `React`_. Listing all
+such libraries is beyond the scope of this document; it's best to just
 try and see!
 
 
 Datashader
 ----------
 
-A major strength of Bokeh is that it copies your data from Python
-directly into the browser, which makes it possible for the user to
-interact with the represented data quickly and responsively, even
-without a live Python process running. However, for plotting millions
-or billions of points, this strength can turn into a liability,
-because current web browsers are very limited in how much data they
-can feasibly work with. Moreover, the visual representation of such
-large datasets is often misleading due to `overplotting`_ and related
-issues.
-
-`Datashader`_ is a separately available Python library that
-pre-renders even the largest datasets into a fixed-size raster image
-that faithfully represents the data's distribution. Datashader
-includes tools and examples showing how to build interactive Bokeh
-plots that dynamically re-render these images when zooming and panning
-in Bokeh, making it practical to work with arbitrarily large datasets
-in a web browser.
+Bokeh lets you pass data from Python directly into a browser so you
+can interact with it quickly and responsively, even without a live
+Python process. However, browser data processing limitations may
+become a problem when plotting millions or billions of points.
+Moreover, visual representation of such large datasets often gets
```

Comment:
Should probably an article like "a visual representation". Or, maybe a little bit easier to read:

> Moreover, creating a visual representation of very large datasets can be misleading because of issues such as overplotting.

Reply:
Ok, I'll re-word.